docs/src/pages/components/select-menu.mdx

```diff
@@ -131,6 +131,41 @@ Available positions:
 </Component>
 ```
 
+## Empty view
+
+It's possible to display a custom empty view instead of options list via `emptyView`, when there are no properties supplied.
+Note that empty view won't be shown when options are being filtered and there are no search results.
```

Comment:
Do/should we have a "no results" view as well?

Reply:
That would be a separate PR and a separate prop.